src/Controller/ContactController.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Controller;
+
+use App\Form\ContactDataType;
+use App\FormData\ContactData;
+use Symfony\Component\Mime\Email;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+
+class ContactController extends AbstractController
+{
+    /**
+     * @Route("/contactpro", name="contactpro")
+     */
+    public function index(Request $request, MailerInterface $mailer): Response
+    {
+        $data = new ContactData();
+        $form = $this->createForm(ContactDataType::class, $data);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $message = (new Email())
+                ->from($data->getEmailAddress())
+                ->to('s.sauvaget41000@gmail.com')
```

Comment:
This should probably be a parameter and in this particular case probably an environment parameter.
See : https://symfony.com/doc/current/configuration.html#configuration-parameters